### PR TITLE
Jan/cleanup

### PR DESCRIPTION
**Pull request by jsulmont**
### Description 

_What behavior does this PR change, and why?_

This PR affects only `Clojure` code in the following way:
* update dependencies
* fix NewTopic constructor call
* readability (format both producer and consumer)

### Author Validation

_Describe the validation already done, or needs to be done, by the PR submitter._
* the example works as advertised [here](https://docs.confluent.io/platform/current/tutorials/examples/clients/docs/clojure.html)

### Reviewer Tasks

_Describe the tasks/validation that the PR submitter is requesting to be done by the reviewer._

### CODE REVIEW

**Comment by ybyzek** [comment]
@jsulmont thanks for the PR!  You may want to retarget the branch to `6.2.0-post` to make the changes immediate.

@shayelkin would you have an opportunity to review the changes?

**Comment by jsulmont** [comment]
@Yeva done (retargeted to 6.2.0-post

**Comment by ybyzek** [comment]
> @Yeva done (retargeted to 6.2.0-post

@jsulmont `jsulmont:jan/cleanup` also requires a rebase against `6.2.0-post`

File: clients/cloud/clojure/src/io/confluent/examples/clients/clj/consumer.clj [hunk 1]
```diff
@@ -11,28 +11,28 @@
...
+      (.putAll {ConsumerConfig/GROUP_ID_CONFIG,                "clojure_example_group"
+                ConsumerConfig/KEY_DESERIALIZER_CLASS_CONFIG   "org.apache.kafka.common.serialization.StringDeserializer"
                 ConsumerConfig/VALUE_DESERIALIZER_CLASS_CONFIG "org.apache.kafka.common.serialization.StringDeserializer"})
       (.load config))))
 
 (defn consumer! [config-fname topic]
   (with-open [consumer (KafkaConsumer. (build-properties config-fname))]
     (.subscribe consumer [topic])
-    (loop [tc 0
+    (loop [tc      0
```
Comment (shayelkin) [inline clients/cloud/clojure/src/io/confluent/examples/clients/clj/consumer.clj]: nit: afaik, unlike some languages Clojure doesn't have an official coding style. Most Clojure code I have previously seen, including everything in the `clojure` github org only has a single space between the binding form and bound expression in the bindings vector, so aligning the init-exprs here looks a bit strange to me. But this is a personal perference.

File: clients/cloud/clojure/src/io/confluent/examples/clients/clj/consumer.clj [hunk 2]
```diff
@@ -11,28 +11,28 @@
 (defn- build-properties [config-fname]
   (with-open [config (jio/reader config-fname)]
     (doto (Properties.)
-      (.putAll {ConsumerConfig/GROUP_ID_CONFIG, "clojure_example_group"
-                ConsumerConfig/KEY_DESERIALIZER_CLASS_CONFIG "org.apache.kafka.common.serialization.StringDeserializer"
+      (.putAll {ConsumerConfig/GROUP_ID_CONFIG,                "clojure_example_group"
```
Comment (shayelkin) [inline clients/cloud/clojure/src/io/confluent/examples/clients/clj/consumer.clj]:
```suggestion
      (.putAll {ConsumerConfig/GROUP_ID_CONFIG                "clojure_example_group"
```
I don't think we need the comma here. Commas are considered whitespace in Clojure, so for maps defined on a single line (e.g. `{:foo foo-val, :bar bar-val}`), it helps readability to add them. For multi-lines one, like here, they're redundant.

**Comment by shayelkin** [comment]
I just have a small nit about indentation. The comma in consumer.clj#L14 is on me, if we could use this PR to undo my own mistakes, I'd appreciate it :)

File: clients/cloud/clojure/src/io/confluent/examples/clients/clj/consumer.clj (hunk 2 above)
Comment (jsulmont) [reply clients/cloud/clojure/src/io/confluent/examples/clients/clj/consumer.clj]: Okay -- I'm using emacs and [cider](https://github.com/clojure-emacs/cider) which has a nice [format function. ](https://github.com/clojure-emacs/cider/blob/master/cider-format.el).

**Review by ybyzek — CHANGES_REQUESTED** [review]
Based on some of the changes in here that changed reference from `6.2.0` to `7.1.0`, I think this PR hasn't been properly rebased to `6.2.0-post` branch?  It requires a rebase (not just branch retarget)

**Comment by ybyzek** [comment]
New PR: https://github.com/confluentinc/examples/pull/1032

**Pull request closed without merging**